ms365_accountcreator/translations/de/LC_MESSAGES/messages.po

```diff
@@ -101,13 +111,18 @@ msgid ""
 "I consent that my st****** email is saved to prevent duplicate Account "
 "creation."
 msgstr ""
-"Ich stimme zu, dass meine st****** E-Mail Adresse gespeichert wird um zu "
-"verhindern, das mehrere Accounts für einen Student angelegt werden "
-"können."
+"Ich stimme zu, dass meine eingegebene st****** E-Mail Adresse verarbeitet"
+" und gespeichert wird um das generierte Passwort zu verschicken und um "
+"sicherzustellen, dass maximal ein Account pro Studi angelegt wird. Mir "
+"ist bewusst, dass ich diese Zustimmung widerrufen und die Löschung der "
+"Daten veranlassen kann."
```

Comment:
```suggestion
"Daten veranlassen kann. (Über die oben genannte support Email Addresse)"
```

Reply:
Das passt, aber die Klammer sollte vielleicht in den Satz (also vor den punkt).

Reply:
ok